00-HTML-CSS-basics/layout2.6_js.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport">
+    <meta charset="utf-8">
+    <link href="layout2.6_js.css" rel="stylesheet" type="text/css">
+<!-- THIS IS JUST TO FIX THE BUG I HAD WITH THE COLOR OF THE IMAGES WHEN THE MOUSE POINTER WAS OVER THE TEXT -->    
+    <script type="text/javascript">
+        function color1() {
```

Comment:
Oh I gotcha! You just created different solutions. Ok!

Please, remember we add a space between method name and parenthesis. `function color1 () {`